gslib/__main__.py

```diff
@@ -82,7 +82,7 @@
 # pylint: disable=g-bad-import-order
 import httplib2
 import oauth2client
-from google_reauth import reauth_creds
+from google_reauth import reauth_creds, errors as reauth_errors
```

Comment:
Based on the surrounding style, I think these imports should be on separate lines